src/utils/command-utils.js

```diff
@@ -0,0 +1,48 @@
+import chalk from 'chalk'
+
+export function getCommmandSingleFlag(flag, commandLine) {
```

Comment:
Maybe we should call this something like `isFlagPresent`? That sounds more meaningful in my opinion.

Reply:
I was thinking about that too, it's better